packages/hooks/__tests__/use-attrs.test.tsx

```diff
@@ -21,7 +20,7 @@ const genComp = (
 ) => {
   return defineComponent({
     inheritAttrs,
-    props: {},
+    props: {} as Record<string, any>,
```

Comment:
Maybe the `props` and `emits` can be written more standardized

Reply:
Seems like this line is not needed.
```diff
- props: {} as Record<string, any>,
```

Reply:
eg.

```ts
props: {
      [TEST_KEY]: String,
    },
    emits: ['click'],
```

Reply:
IMO, in order to test `attrs`, we shouldn't explicitly declare it on component `props` and `emit`. If we do as above, the `[TEST_KEY]` will be treated as `props` instead of `attrs`. Is my understanding correct?

Reply:
you are right